libraries/AP_HAL_ChibiOS/hwdef/JFB100/defaults.parm

```diff
@@ -0,0 +1,4 @@
+
+# JFB-100 does not have hw flow control on radios
```

Comment:
does the connector have flow control pins? If not then leave them out of the hwdef.dat

Reply:
I changed it to a new default parameter.